snprc_ehr/resources/queries/snprc_ehr/ExportNewAnimalData.sql

```diff
@@ -0,0 +1,43 @@
+/********************************************************
+  Query will feed the export ETL back to CAMP.
+      CampNewAnimalData
+
+
+ srr 07.07.2020
+ ********************************************************/
+
```

Comment:
Did you leave cage out for a reason?

Reply:
No.  It's in the CAMP side create table statement but I missed it in the ETL query.  I'll add it and also a processed date-time column to the CAMP side table.